test/integration/timeout.integration.test.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * © Copyright IBM Corporation 2021. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const assert = require('assert');
+const sinon = require('sinon');
+
+const { CloudantV1 } = require('../../index.ts');
+const { CouchdbSessionAuthenticator } = require('../../index.ts');
+const {
+  BasicAuthenticator,
+  IamAuthenticator,
+  NoAuthAuthenticator,
+} = require('ibm-cloud-sdk-core');
+
+const TESTCASE_TIMEOUT = 10000; // (10s)
```

Comment:
Do these stubbed tests really need such a long timeout?

Reply:
No, I removed this setting: 0fda4a4 
The default jest timeout will be now active that is 5s.